### PR TITLE
miksyuk/registration/password-bullet

### PR DESCRIPTION
**Pull request by Miksyuk**
added test for Registration page "password and confirn password replaced by bullet"

### CODE REVIEW

File: test/expected/registration.js [hunk 1]
```diff
@@ -23,5 +23,6 @@ export default {
   confirmEmailPlaceholder: 'Confirm Email *',
   confirmPasswordPlaceholder: 'Confirm Password *',
   errorUserExists: 'User with this email already exists',
-  focusHighlightColorLastName: 'rgba(0,123,255,0.25)'
+  focusHighlightColorLastName: 'rgba(0,123,255,0.25)',
+  passwordDisplay: 'block'
```
Comment (artsenius) [inline test/expected/registration.js]: This is incorrect expected. It's not related to bullets.

File: test/specs/reg/registration/general.js [hunk 2]
```diff
@@ -177,4 +177,16 @@ describe('Password', function () {
     assert.equal(fontFamily, exp.fontFamily);
   });
 
+  it('Password is replaced by bullets', function () {
+    $(sel.password).addValue(regData.pass);
+    let actual = $(sel.password).getCSSProperty('display').value;
```
Comment (artsenius) [inline test/specs/reg/registration/general.js]: you should look for this style
`input[type="password" i] {
    -webkit-text-security: disc !important;
}`

File: test/specs/reg/registration/general.js [hunk 3]
```diff
@@ -177,4 +177,16 @@ describe('Password', function () {
...
+    assert.equal(actual, exp.passwordDisplay);
+  });
+
+  it('Confirm Password is replaced by bullets', function () {
+    $(sel.confirmPassword).addValue(regData.pass);
+    let actual = $(sel.confirmPassword).getCSSProperty('display').value;
```
Comment (artsenius) [inline test/specs/reg/registration/general.js]: same here

File: test/specs/reg/registration/general.js (hunk 2 above)
Comment (Miksyuk) [reply test/specs/reg/registration/general.js]: done

File: test/expected/registration.js (hunk 1 above)
Comment (Miksyuk) [reply test/expected/registration.js]: done

File: test/specs/reg/registration/general.js (hunk 3 above)
Comment (Miksyuk) [reply test/specs/reg/registration/general.js]: done

Comment (Miksyuk) [reply test/specs/reg/registration/general.js]: @artsenius check one more time pls

**Pull request merged**